packages/by-name/cloud-api-adaptor/package.nix

```diff
@@ -31,19 +32,28 @@ in
 
 buildGoModule rec {
   pname = "cloud-api-adaptor";
-  version = "0.9.0";
+  version = "0.10.0";
+
+  src = applyPatches {
```

Comment:
Should we use `applyPatches` here? This is only useful if there are other packages depending on the source of this one, which I can't think of in the case of CAA?

Reply:
I need to either apply patches before the `sourceRoot`, or apply patch flags. I find the former easier to reason about, and it allows cherry-picking patches from upstream that modify outside files - is there a downside to using `applyPatches`?

Reply:
Not really, it's just not something you see very often in Nix. I agree with your reasoning though and think it makes sense in that case.